packages/flutter/lib/src/material/segmented_button.dart

```diff
@@ -335,11 +340,18 @@ class SegmentedButton<T> extends StatelessWidget {
             child: label,
           );
 
+      final Widget buttonWithTooltip = (segment.tooltip != null)
```

Comment:
are the parentheses needed?

Reply:
No, I've completely missed the fact they are not obligatory in Dart. I don't have much experience with contributing, how shuld this be fixed now, when it's merged already?